09-features.md

```diff
@@ -39,6 +39,7 @@ The Context column decodes as follows:
 | 16/17 | `basic_mpp`                      | Node can receive basic multi-part payments                | IN9      | `payment_secret`  | [BOLT #4][bolt04-mpp]                 |
 | 18/19 | `option_support_large_channel`   | Can create large channels                                 | IN       |                   | [BOLT #2](02-peer-protocol.md#the-open_channel-message) |
 | 20/21 | `option_anchor_outputs`          | Anchor outputs                                            | IN       | `option_static_remotekey` | [BOLT #3](03-transactions.md)         |
+| 22/23 | `option_pubkey_routing`          | Public key routing                                        | IN       | `var_onion_optin` | [BOLT #4](04-onion-routing.md#tlv-payload-format)         |
```

Comment:
Shouldn't the flags be `IN9`?
It feels like you need to set it to required on your invoice when you're using it, otherwise senders will not correctly include the `outgoing_node_id` field in the onion.

Reply:
The idea is that senders recognize the all-zeroes channel id and use this as a cue to switch to pubkey routing. But for older senders this will just lead to failures that an invoice feature bit can prevent. Added the 9.